Sources/FluentSQL/Deprecated.swift

```diff
@@ -24,5 +24,18 @@ extension SchemaBuilder {
     }
 }
 
+extension SchemaBuilder {
+    /// - warning: Deprecated.
+    @available(*, deprecated, renamed: "unique(on:)")
+    public func addIndex<T>(to: KeyPath<Model, T>, isUnique: Bool = false) {
+        return unique(on: to)
+    }
+    /// - warning: Deprecated.
+    @available(*, deprecated, renamed: "unique(on:_:)")
+    public func addIndex<A, B>(to: KeyPath<Model, A>, _ and: KeyPath<Model, B>, isUnique: Bool = false) {
+        return unique(on: to, and)
+    }
+}
+
 @available(*, deprecated, renamed: "SQLDatabase")
 public typealias SchemaSupporting = SQLSupporting
```

Comment:
Shouldn't the warning message say it was renamed to `SQLSupporting`, not `SQLDatabase`?